pkg/virt-api/validating-webhook/validating-webhook_test.go

```diff
@@ -809,6 +810,47 @@ var _ = Describe("Validating Webhook", func() {
 			Expect(len(causes)).To(Equal(1))
 			Expect(causes[0].Field).To(Equal("fake[0].bootorder"))
 		})
+
+		It("should reject invalid SN characters", func() {
```

Comment:
Do we have a test where we see that it accepts valid values?

Reply:
@rmohr Sorry, haven't gotten back to this.  I've added the unit test, that's easy enough; the trick is that the functional test uses the Cirros image that doesn't have some of the extras including dev/disk-by-id and a new enough version of lsblk to query the serial number.  I tried some other options like hdparm which exists, but again with cirros and the virtio driver the result is an ioctl error.  I even thought I should be able to extract it from proc somewhere but no luck with that.

My plan was to load up a Fedora image and verify things work, then consider if we need to propose adding that to the functional tests or not.  If you or others have other ideas/options let me know; otherwise I'll pick back up on this tomorrow and see if I can find another option.

Reply:
@j-griffith sorry for the late response. That works for cirros:

```
# is the serial somewhere?
$ grep sv123ggfg /sys/* -r 2>/dev/null
/sys/devices/pci0000:00/0000:00:02.3/0000:04:00.0/virtio2/block/vdb/serial:sv123ggfg
# ok that gives us the serial but no newline, that's annoying with expecters
$ find /sys -type f -regex ".*/block/vdb/serial" | xargs cat
sv123ggfg$ 
# now with newline
$ find /sys -type f -regex ".*/block/vdb/serial" | xargs cat && echo
sv123ggfg
```

Reply:
@j-griffith btw. used your PR to set the serial `sv123ggfg`. Works great. :+1: